src/controllers/update/overseas.entity.query.controller.ts

```diff
@@ -0,0 +1,34 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+import { getApplicationData, setExtraData } from "../../utils/application.data";
+import { OeNumberKey } from "../../model/data.types.model";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.OVERSEAS_ENTITY_QUERY_PAGE}`);
+
+    return res.render(config.OVERSEAS_ENTITY_QUERY_PAGE, {
```

Comment:
We need to fetch `oeNumber` from the session and passed to the render. Already described [here](https://github.com/companieshouse/overseas-entities-web/blob/main/docs/Project%20Structure%20and%20Code%20Style.md#the-controller)

Reply:
done, let me know if that's ok as well